### DISCUSSION TITLE
[PNP FILTER WEBPART] USING MICROSOFT SEARCH - How to show sitecolumns or metadata from list/library

### DISCUSSION THREAD

**Discussion by hsamysaab91** [microsoft-search/pnp-modern-search#2752] [Q&A]
Hi guys,

I'm not sure if it's a bug or not. But on the filter webpart of a connected result source (using pnp), I can't find a way of adding my refinablestring or any managed property besides the one proposed. 
I tried resource.[internalNameOfMyColumn] I tried to put refinablestring06 directly as well. Nothing works.

Can you please help me ? 

![image](https://user-images.githubusercontent.com/38861691/217238494-48fa695c-c191-4ad6-965d-c62eff9bde68.png)

**Comment by wobba** [comment]
Did you follow the tutorial on https://microsoft-search.github.io/pnp-modern-search/scenarios/page-with-filters/?

**Reply by hsamysaab91** [reply]
Hi Wobba,

Yes I did.  I created a refinablestring06 using a sitecolumn called "department". I'm not able to find it on the search result webpart, so I added it by typing it directly to the "selected columns".

![image](https://user-images.githubusercontent.com/38861691/217261881-9ac95a91-b367-4524-8790-fae3f991f86e.png)


Then I went to the filter webpart, and I can't see any of the added column on the dropdown.

![image](https://user-images.githubusercontent.com/38861691/217261622-a0e21d94-1e77-4b02-94de-9f5381de780f.png)

**Reply by wobba** [reply]
You need to type RefinableString06 manually in the filter part as well, and then you get values. When using the Graph provider this is a bit trickier and we could look at improving it.

![image](https://user-images.githubusercontent.com/51104/217346405-174af0b9-1a8c-4632-8ceb-a7931cfa34fc.png)
